execution_engine/src/runtime_context/mod.rs

```diff
@@ -1193,9 +1203,6 @@ where
         let package_hash_key = Key::from(package_hash);
         self.validate_key(&package_hash_key)?;
         let contract_package: Package = self.read_gs_typed(&Key::from(package_hash))?;
-        if !self.is_authorized_by_admin() {
-            self.validate_uref(&contract_package.access_key())?;
-        }
```

Comment:
are we sure we don't need this?

Reply:
Well we don't need the admin check I think, I will double check

Reply:
sure...but I read this to say that on a closed network an administrator can upgrade any contract (ie the code indicates that the 1x logic skips access validation if is_authorized_by_admin == true).

I think that our go forward on this PR does not make such an allowance and apparently should.

Reply:
This is just a check to retrieve the contract package, we check access by the access uref, but I dont think its needed since the field is no longer on the package